### DISCUSSION TITLE
utools 版 flomo

### DISCUSSION THREAD

**Discussion by extrastu** [flomoapp/3rd-party-tools#21] [分享创造]
# Flomo for utools 
> 使用官方提供API写的Flomo小插件

## 预览

![书写memo](https://tva1.sinaimg.cn/large/0081Kckwgy1glz2xu0t86j30tp0gowf8.jpg)

![填写API](https://tva1.sinaimg.cn/large/0081Kckwgy1glz2yk9j0gj30tp0gojs9.jpg)

### v0.0.1
- 初始版本提交

### v0.0.2（当前版本）
- 默认标签 ```#utools``` 放开，用户可以自定义
- 由于没有仔细看utools说明导致v0.0.1版本不可热更新到本版本，后续可以支持热更新从此版本开始
- ```API modal``` 支持显示已填```API``` 

....

## 使用说明
> 使用官方API，不用担心安全问题
- 默认增加 `#utools`标签，方便管理
- utools 输入 `flomo` / `memo` 即可 进入
- 点击设置按钮输入API 地址即可愉快的记录memo

## 更多

如果喜欢可以使用[https://flomoapp.com/register2/?NjM0](https://flomoapp.com/register2/?NjM0])注册

##PS

不支持传文件，需要自己下载utools ，然后在插件中心就可以看到flomo了
![](https://tva1.sinaimg.cn/large/0081Kckwgy1gm3ie0nwecj30qe0hd75f.jpg)

**Comment by zemon99207** [comment]
如果能按回车键直接发送就好了，用鼠标移上去点的话操作上不太连贯

**Reply by whitewatercn** [reply]
+1

**Reply by ovechou** [reply]
+1

**Reply by woodgear** [reply]
+1

**Comment by whitewatercn** [comment]
用着不太对劲呀，一直在发送中状态
（API填对了，网络状态正常

**Reply by extrastu** [reply]
是的，目前商店版本有问题，可以下载git上我放的
